version.py

```diff
@@ -34,7 +34,9 @@
 # new level means in api_docs/changelog.md, as well as "**Changes**"
 # entries in the endpoint's documentation in `zulip.yaml`.
 
-API_FEATURE_LEVEL = 312  # Last bumped for adding 'realm_export_consent' event type.
+API_FEATURE_LEVEL = (
+    313  # Last bumped for accepting anonymous groups for create_multiuse_invite_group.
+)
```

Comment:
Better to write a shorter comment here than let it get line-wrapped. These comments are just to help avoid forgetting to bump this when there are merge conflicts.

Reply:
Fixed!